README.md

```diff
@@ -1 +1,18 @@
-# hw01
\ No newline at end of file
+# hw01
+
+# Greetings!  
+
+My name is **Maya Krasovsky**.
+
+![*This is me outisde, taking in the sunshine*](https://user-images.githubusercontent.com/63366598/79288701-0f46ad00-7e8d-11ea-9f5b-d4be6c0efdd7.jpg)
```

Comment:
Using a link is definitely a great way to add images. I was initially struggling with adding an image too. I was using RStudio Server and the way I used to add my image was to upload it to the server and then simply add the name of my image. I'll try using the method of using a link next time too.

Reply:
@Ojaswi1 that's the best approach if your image is not already posted somewhere online